t/cli/test_yaml_config_variables.sh

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bash
+
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+. ./t/cli/common.sh
+
+# check supported environment variables in apisix.yaml
+
+yaml_config_variables_clean_up() {
+    git checkout conf/config.yaml
```

Comment:
We can use the original clean up like:
https://github.com/apache/apisix/blob/380f762f30fb845893c1f46393c4de6e57b808a6/t/cli/test_serverless.sh#L21-L25

Reply:
Fixed. Please review it again.